server/models/users.js

```diff
@@ -36,11 +36,15 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
     },
     username: {
```

Comment:
Do we still need `username` in the `Users` table?

Reply:
No